Modules/Scripted/SegmentStatistics/SegmentStatisticsCalculators/SegmentStatisticsCalculatorBase.py

```diff
@@ -0,0 +1,201 @@
+import vtk
+import qt
+import slicer
+
+
+class SegmentStatisticsCalculatorBase(object):
+  """Base class for statistics calculators operating on segments.
+  Derived classes should specify: self.name, self.keys, self.defaultKeys
+  and implement: computeStatistics, getMeasurementInfo
+  """
+
+  @staticmethod
+  def initCodedEntry(codeValue, codingScheme, codeMeaning, returnAsString=False):
+    entry = slicer.vtkCodedEntry()
+    entry.SetValueSchemeMeaning(codeValue, codingScheme, codeMeaning)
+    return entry if not returnAsString else entry.GetAsString()
+
+  @staticmethod
+  def getDICOMTriplet(codeValue, codingSchemeDesignator, codeMeaning):
```

Comment:
Why not use a single method for getting a coded entry in any type? (merging initCodedEntry and getDICOMTriplet)

    def createCodedEntry(codeValue, codingScheme, codeMeaning, type='vtk')

type could be 'vtk', 'json', and 'string'